addons/move-to-top-layer/addon.json

```
@@ -0,0 +1,31 @@
+{
+  "name":"Move Sprite to Top Layer",
```

Here's an opinionated suggestion:
```suggestion
  "name":"Move sprite to front",
```
Also, to be consistent with other addons, we don't need to use title case on the name.

---

I think it's more obvious what "top layer" does, so I'd suggest "Move sprite to top layer".

---

Scratch calls it the "front layer".

---

> Scratch calls it the "front layer".

Perhaps `Move sprite to front layer`, then?

---

I'm going to change it to `Move sprite to front layer` as soon as I'm able to push into the PR branch (currently cannot)